### ISSUE TITLE
Client methods undefined

### ISSUE THREAD

**Issue by mrh-is** [contentful/contentful-management.rb#102]
Some of the`Client` methods aren't available. Weirdly, it looks like all the `attr`s and the methods marked `# @private` are defined, but the public ones are not. My testing code below:

```ruby
require 'contentful/management'

client = Contentful::Management::Client.new('<ACCESS_TOKEN>')
puts client            # exists, everything is cool
client.access_token    # this exists
client.api_version     # works
spaces = client.spaces # 💥
```

```ruby
puts client.methods

delete
configuration
version
version=
post
user_agent
get
put
protocol
default_locale
access_token
request_headers
logger
base_url
organization_id
zero_length
content_type_id
dynamic_entry_cache
api_version
gzip_encoded
default_configuration
setup_logger
update_dynamic_entry_cache_for_spaces!
update_dynamic_entry_cache_for_space!
update_dynamic_entry_cache!
register_dynamic_entry
execute_request
clear_headers
content_type_id=
organization_id=
authentication_header
api_version_header
organization_header
version_header
content_type_header
zero_length_header
accept_encoding_header
zero_length=
dynamic_entry_cache=
# regular Object methods
# ...
```

**Comment by dlitvakb** [comment]
Hey @mrh-is,

Can you confirm what version are you using?

Client methods are available from version `1.0` onwards, while `0.x` uses a different approach.

If you're on `0.x`, methods are as follows:

```ruby
# Get entries
Contentful::Management::Entry.all(params)
Contentful::Management::Entry.find(id)

# Get spaces
Contentful::Management::Space.all(params)
Contentful::Management::Space.find(id)
```

All Contentful resources follow the same pattern

Hope that helps

**Comment by mrh-is** [comment]
@dlitvakb Yep, that was it! I'm also using `contentful-importer`, which relies on [`'~> 0.7.0'`](https://github.com/contentful/contentful-importer.rb/blob/master/contentful_importer.gemspec#L24). Thanks for the pointer!
